[XSUP-54971] Cloudflare WAF content pack default parsing rule error. (#41003)
demisto/content: Packs/CloudflareWAF/ReleaseNotes/1_0_31.md

#### Parsing Rules

##### Cloudflare WAF Parsing Rule

> - Fixing the parsing rule error for epoch timestamp – added casting to integer when dividing nanoseconds.

Reviewer: ```suggestion
- Fixed the parsing rule error for the epoch timestamp by adding casting to an integer when dividing nanoseconds.
```
